blackbox/docs/admin/privileges.rst

```diff
@@ -109,15 +121,33 @@ and functions of the ``doc`` schema::
     on a schema level, CrateDB takes the schema name provided without further
     validation.
 
-This statement will deny ``DQL`` privilege to user riley on the ``doc`` schema
-table ``doc.accounting``. However, user riley, will still have ``DQL``
-privilege on all the other tables of the ``doc`` schema.
-
 .. NOTE::
 
     Privileges can be managed on all schemas and tables of the cluster,
     except the ``information_schema``.
 
+Views are on the same hierarchy with tables, i.e. a privilege on a view
+is gained through a ``GRANT`` on either the view itself, the schema the view
+belongs to, or a cluster-wide privilege. Privileges on tables which are
+referenced in the view do not grant any privileges on the view itself. On the
+contrary, even if the user does not have any privileges on a view's referenced
+tables but on the view itself, the user can still access the tables through
+the view. For example::
```

Comment:
Shouldn't we mention the owner concept here as well?

Reply:
We could, but I am not sure, as it is kind of an implementation detail that is not necessarily important for the user.

Reply:
I don't think the owner is an implementation detail. It's important for users to understand the semantics. And it has a huge effect on how the privilege system behaves.

But maybe we could trim this down here a bit and link to the `ddl/views.rst` section instead.